vcloud_plugin_common/__init__.py

```diff
@@ -239,7 +251,8 @@ def _subscription_login(self, url, username, password, token, service,
         atexit.register(vca.logout)
         return vca
 
-    def _ondemand_login(self, url, username, password, token, instance_id):
+    def _ondemand_login(self, url, username, password, token, instance_id,
+                        session_token, org_url):
         """
```

Comment:
set last params default value to None, and skip None in calls before this place.